cockpit_api/src/main/java/com/cockpit/api/service/jiragateway/JiraGatewayService.java

```diff
@@ -304,4 +173,201 @@ public void setTotalNbOfUserStoryForEachSprintOfEachProject() {
             }
         }
     }
+
+    @Scheduled(initialDelay = 10 * ONE_SECOND, fixedDelay = 10 * ONE_MINUTE)
+    public void deleteJiraProjects() throws UnirestException {
+        List<Jira> jiraProjectList = jiraRepository.findAllByOrderById();
+        for(Jira jira : jiraProjectList){
+            HttpResponse<JsonNode> jiraProject = Unirest.get(jiraUrl+"/rest/api/3/project/"+jira.getJiraProjectKey())
```

Comment:
It could be better to get the list of sprints with one api call and and compare them with found ones in the database.
As we discussed before, calling Jira GET for 100+ times(for example 20 projects with 5 sprints for each) in a for function is dangerous for Jira server and it takes long time.
Why not use this url : https://tdf.atlassian.net/rest/agile/1.0/board/{boardId}/sprint, just need to make a for loop with  scrum boards.
One possible solution for example:
1. Call .../rest/agile/1.0/board?type=scrum to get the scrum borad Ids as you did for getJiraSprints();
2. Collect the boardId for revevied data.
3. Call https://tdf.atlassian.net/rest/agile/1.0/board/{boardId}/sprint to receve sprints
4; Get all sprints from database
5 Compare and delete when necessary